apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.html.ts

```diff
@@ -28,6 +28,75 @@ export const AssessmentRowSchema = AssessmentSchema.merge(
 });
 type AssessmentRow = z.infer<typeof AssessmentRowSchema>;
 
+export function InstructorInstanceAdminLti13NoInstances({
+  resLocals,
+  ltiInstances,
+  institutionName,
+}: {
+  resLocals: Record<string, any>;
+  ltiInstances: Lti13Instance[];
+  institutionName: string;
+}): string {
+  const content = html`
+    <div class="card mb-4">
+      <div class="card-header bg-primary text-white d-flex align-items-center">
+        <h1>Integrations with other learning systems</h1>
+      </div>
+      <div class="card-body">
+        <p>
+          PrairieLearn supports integrations with learning systems to pass status and grades between
+          them. No integrations have been configured yet for this course instance, but when that
+          happens you can configure them there.
+        </p>
+
+        ${ltiInstances.length === 0
+          ? html`
+              <p>
+                No integrations are present with ${institutionName}. Reach out to your learning
+                system admins to explore how they might integrate with PrairieLearn. The
+                <a target="_blank" href="https://prairielearn.readthedocs.io/en/latest/lti13/"
+                  >administrators LTI 1.3 configuration documentation</a
+                >
+                is a good starting point for them.
+              </p>
+            `
+          : html`
+              <p>
+                The following ${institutionName} learning systems are already integrated with
```

Comment:
I did several double takes reading things like "The following Default learning systems". IMO we should not be using the institution name here. Folks know what institution they're from. We also have hundreds of legacy courses not associated with any specific institution.

If we need to refer to an institution, I think just "your institution" is fine:

- "No integrations have been configured for your institution"
- "The following learning systems from your institution are already integrated..."

Reply:
Reverted to "your institution" language in 11fd31e589